samples/meetings-token-app/csharp/ClientApp/package.json

```diff
@@ -30,8 +30,8 @@
   },
   "scripts": {
     "clean": "npx shx rm -rf build",
-    "start": "react-scripts start",
-    "build": "react-scripts build",
+    "start": "react-scripts --openssl-legacy-provider start",
+    "build": "react-scripts --openssl-legacy-provider build",
```

Comment:
Why we have added it, share the RCA for it.

Reply:
I built and tested my solution locally (http://localhost:3978/), but it does not work. So that is the reason I have added this RCA.